src/components/results/dimension-based-results/dimension/index.js

```diff
@@ -41,11 +41,11 @@ export default class Dimension extends Component{
         <div class={style.main} style={`background: ${Color.rgba(color, 8.5)};border-left: 5px solid ${color};`}>
           <div class={style.side}>
             <p class={style.icon}>
-              <img src={type.badge.image_medium} alt={type.name} />
+              <img src={type.badge.image_medium} alt="" role="presentation" ariahidden="true" />
```

Comment:
Shouldn't the alt still be in there so that people know what it is if they can't see or if their internet doesn't load it?

Reply:
Normally, you'd be right, but because the Type name is repeated below, this image is just decoration, so with the alt text included, it is reading both the Alt text and the regular text for screen readers. This change removes the Alt text so it is not read twice. Accessibility and all that.